gadgetchains/CodeIgniter4/FD/2/chain.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace GadgetChain\CodeIgniter4;
+
+class FD2 extends \PHPGGC\GadgetChain\FileDelete
+{
+    public static $version = '<= 4.3.7';
```

Comment:
@cfreal stupid question, but what would be the proper way to signal that a given chain is working on a given version, and hasn't been patched yet? Because `'<= 4.3.7'` conveys to me that this chain works on all versions equal of below `4.3.7`, but since `4.3.7` is the current latest one, odds are that in 6 months, when new versions will be released, it'll still work on them.

Reply:
Hi,

One should use `+` to indicate that it may work for superior versions as well. With your example:  `<= 4.3.7+`. (I'll try to find the issue in which I agreed with this model when I can.) Open to suggestions though! 

Charles

Reply:
Maybe add a note about this in the [documentation](https://github.com/ambionics/phpggc#contributing) then <3

Reply:
See [commit](https://github.com/ambionics/phpggc/commit/ae02526f32489c56847282aefa8c4b36b1f7e4e5).